src/genie/libs/parser/iosxe/show_ipv6_protocols.py

```diff
@@ -0,0 +1,958 @@
+''' show_ipv6_protocols.py
```

Comment:
Sorry, I missed this before: Please move the contents of this file over into `iosxe/show_protocols.py` (similar to how you incorporated your code into `iosxe/show_eigrp.py`)